pkg/proxy/proxy.go

```diff
@@ -154,6 +211,11 @@ func defaultPolicies() []config.Policy {
 					Endpoint: "/ocs/",
 					Backend:  "http://localhost:9140",
 				},
+				config.Route{
+					Type:     config.QueryRoute,
+					Endpoint: "/remote.php/?preview=1",
```

Comment:
could `type Route`, instead of defining the query string as part of the endpoint and potentially getting cluttered, have a `matcher` or `regexp` field?

Reply:
That opens one problem I had.
How do I get the value? Currently the structure only holds information about the endpoint and the type. If we would replace the endpoint with the route than how could we efficiently retrieve the default route?